src/index.css

```diff
@@ -72,4 +72,12 @@ textarea{
 }
 .fa-bars{
     font-size: 25px;
+}
+
+/* -----------logout-model---------------- */
+
+.logout-model{
+    position: fixed;
+    top:50px;
```

Comment:
dont appy the static values for centering the element

Reply:
okk mr-hollowman